apps/core/scenes/search/SearchContext.js

```diff
@@ -163,10 +153,23 @@ export default class SearchContextProvider extends React.Component<
         clearLocation: this.clearLocation,
         addLocation: this.addLocation,
         setLocation: this.setLocation,
+        setStateFromQueryParams: this.setStateFromQueryParams,
       },
     };
   }
 
+  parseFields = (params: Object): Object => {
```

Comment:
You should be able to give proper types for params and the return type as well

Reply:
`Flow` won't let me as we are spreading `{ [key]: value }` so it does not recognize the spreaded keys (d'uh)